package.json

```
@@ -33,6 +33,10 @@
   },
   "homepage": "https://github.com/nmaves/passport-slack-oauth2#readme",
   "dependencies": {
-    "passport-oauth2": "^1.7.0"
+    "passport-oauth2": "file:../passport-oauth2"
```

I think you want to revert this line.
```suggestion
    "passport-oauth2": "^1.7.0"
```

---

@nmaves yes I do! However, atm, the required changes haven't been pulled into passport-oauth2. Do you think you could take a look over there and try to nudge that along? I don't know the best way to get attention on it.